src/main/kotlin/com/kotlin/boardproject/domain/post/repository/LikePostRepository.kt

```diff
@@ -0,0 +1,10 @@
+package com.kotlin.boardproject.domain.post.repository
+
+import com.kotlin.boardproject.domain.post.domain.BasePost
+import com.kotlin.boardproject.domain.post.domain.LikePost
+import com.kotlin.boardproject.domain.user.domain.User
+import org.springframework.data.jpa.repository.JpaRepository
+
+interface LikePostRepository : JpaRepository<LikePost, Long> {
+    fun findByUserAndPost(user: User, post: BasePost): LikePost?
```

Comment:
🚫 **[ktlint]** <standard:function-signature> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should start on a newline